airflow/config_templates/__init__.py

```diff
@@ -16,3 +16,4 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from . import airflow_local_settings
```

Comment:
@rhwang10 whats this addition doing?

Reply:
I think this was an addition to get the dev setup up and running- had to do with how the PYTHONPATH was exported in the dockerfile- we can remove it